api/apps/api/src/modules/access-control/projects-acl/project-acl.controller.ts

```diff
@@ -0,0 +1,80 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Get,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  ForbiddenException,
+  Delete,
+  Patch,
+  Body,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ProjectAclService } from './project-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UsersProjectsApiEntity } from '@marxan-api/modules/projects/control-level/users-projects.api.entity';
+import { UpdateUserRoleinProject } from './dto/update-user-role-project.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Projects Users')
+@Controller(`${apiGlobalPrefixes.v1}/projects`)
+export class ProjectAclController {
+  constructor(private readonly projectAclService: ProjectAclService) {}
+
+  @Get(':projectId/users')
+  async findUsersInProject(
+    @Param('projectId', ParseUUIDPipe) projectId: string,
+    @Req() req: RequestWithAuthenticatedUser,
+  ): Promise<UsersProjectsApiEntity[]> {
```

Comment:
Controller should never operate on Entities - dto/entity/pojos have different purpose and even tho it may seem overengineering, we don't want to force REST API be that coupled to entities.